### ISSUE TITLE
[MHV MR / Accessibility] Instructions for downloading files appear after the download file menu

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#121932]
### 📝 User story
As a user, I want instructions related to printing and downloading to appear before the print/download control, so that I am prepared to interact successfully with the control.

### 📝 Problem statement
"What to know before you download" instructions appear after the print/download control in the focus and content order.

### 📝 Details
<details><summary>Links, screenshots, steps to recreate</summary>

#### 🌌 Supporting screenshots, videos, code snippets

---

##### 🔗 From [Medical Records > Care summaries > ADHC Consult Results](https://staging.va.gov/my-health/medical-records/summaries-and-notes/811)

<img width="674" height="348" alt="Screenshot showing the print or download button followed by the instructions before you download control" src="https://github.com/user-attachments/assets/88542b3c-e997-4933-87ef-b41422123939" />

---

#### 👣 Steps to recreate
1. Navigate to the Medical Records > Care summaries and notes list page.
2. Navigate to a details page (e.g., “ADHC Consult Results”).
3. Use the Tab key to navigate to the “Print or download” control.
4. Notice the “What to know before you download” control after the “Print or download” control.
5. Use the Tab key to navigate to the “What to know before you download” control.

</details>

---

### 🔗 Point of contact
Sarah Horton (@sarahhortonadhoc)

### ⚠️ Severity level
- 2, Serious. Should be fixed in 1-2 sprints post-launch.

#### Accessibility Standards
- [Guideline 1.3 Adaptable: Create content that can be presented in different ways (for example simpler layout) without losing information or structure](https://www.w3.org/WAI/WCAG21/Understanding/adaptable)
  - [1.3.2 Meaningful Sequence (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence.html)
- [Guideline 2.4 Navigable: Provide ways to help users navigate, find content, and determine where they are](https://www.w3.org/WAI/WCAG21/Understanding/navigable)
  - [2.4.3 Focus Order (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)

#### Experience Standards
- [Comprehension: User easily understands the product](https://design.va.gov/about/experience-standards/#category-comprehension)
  - [User has enough information to complete a task](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
- [Consistency: User has a predictable and unchanging experience across all VA.gov and VA Mobile App services, tools, and pages](https://design.va.gov/about/experience-standards/#category-consistency)
  - [User experiences predictable interactions with components and patterns](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components-and-patterns)

### 𝌣 Proposed solution or next steps
Move the “What to know before you download” control before the “Print or download” control.

### ✅ Acceptance criteria
- [ ] The system displays the “What to know before you download” control before the “Print or download” menu.
- [ ] The system provides the “What to know before you download” control and associated content before the “Print or download” menu code and content in the code order.
- [ ] The user can use the Tab key to navigate to the “What to know before you download” control and then to the “Print or download” menu.

### ✅ Possible tasks
- [ ] Review, refine and acknowledge feedback.
- [ ] Update ticket information (assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

#### General resources
- USWDS Docs: [USWDS Documentation](https://designsystem.digital.gov/)
- VADS Docs: [VADS Experience Standards](https://design.va.gov/about/experience-standards/)
- VADS Docs: [VADS Components Documentation](https://design.va.gov/components/)
- VADS Docs: [VADS Components Storybook](https://design.va.gov/storybook/?path=/docs/about-introduction--docs)
- VA Platform Docs: [Accessibility defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric)
- VA Github Docs: [List of common issue tickets](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/accessibility/common-issues) you can reference, copy, and add to.
- W3C Docs: [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/)
- W3C Docs: [WCAG 2.2 Understanding Docs](https://www.w3.org/WAI/WCAG22/Understanding/)

**Comment by steveg-IAT** [comment]
@jonathanjnelson  just adding you as the assignee for this ticket as you are working with @sarahhortonadhoc  directly.

**Comment by jonathanjnelson** [comment]
Yes @sarahhortonadhoc! Thanks for calling that out.

**Comment by jonathanjnelson** [comment]
@GovNapoleon Thanks for your work on this. Part of this work is also moving the pattern towards the bottom of the screen to be more consistent with medication. Please let me know if you have any questions and we can do a screen share. 

<img width="5321" height="5602" alt="Image" src="https://github.com/user-attachments/assets/696fe862-c3a2-424a-a49e-0eb0eb0641f4" />

**Comment by GovNapoleon** [comment]
Health Conditions detail Page

<img width="2493" height="1294" alt="Image" src="https://github.com/user-attachments/assets/cb1a322d-46fe-4d6c-baa8-510ce418094a" />

**Comment by sarahhortonadhoc** [comment]
@GovNapoleon it would be good to establish a visual design pattern to visually separate the print/download section from the content. @jonathanjnelson I noticed that Medications applies a top border to the section — would you like to add that on Medical Records?

<img width="675" height="302" alt="Screenshot of print/download feature with horizontal rule after details and before feature" src="https://github.com/user-attachments/assets/ff0f78af-dec5-4709-ac8e-0d4cc7d65536" />

**Comment by GovNapoleon** [comment]
@sarahhortonadhoc this is Vaccine Details Page. I can follow this pattern for all of the other pages.

<img width="2513" height="714" alt="Image" src="https://github.com/user-attachments/assets/08a07e1a-fa96-4761-8a3a-fc6faf175714" />

**Comment by jonathanjnelson** [comment]
Thanks @GovNapoleon. Great question. 

The print/download pattern should be below the pagination (if shown) and above the line (which should be there) 

<img width="571" height="416" alt="Image" src="https://github.com/user-attachments/assets/f34f0ba9-dbd3-488d-9455-8337685f9e9b" />

**Comment by davidkoger** [comment]
confirmed in staging

**Comment by sarahhortonadhoc** [comment]
@GovNapoleon can you confirm that this issue has been addressed so I can close the ticket? Thank you!
